### ISSUE TITLE
No description for sim-specific Preferences

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/membrane-transport#367]
Related to https://github.com/phetsims/phet-info/issues/241… 

Membrane Transport is missing description for sim-specific Preferences.  In A11y View:

<img width="964" height="523" alt="Image" src="https://github.com/user-attachments/assets/192bfb30-cdf0-4c67-a10a-f5382105b68f" />

**Comment by brettfiedler** [comment]
@jessegreenberg or @samreid , can you implement and add placeholder content here?

**Comment by jessegreenberg** [comment]
If this can wait a few days, lets try to do it in https://github.com/phetsims/joist/issues/1022.

**Comment by terracoda** [comment]
I added these to the Yaml file yesterday.

**Comment by terracoda** [comment]
We need context responses, though, and there are differences between keyboard and mouse delivery.

<img width="414" height="162" alt="Image" src="https://github.com/user-attachments/assets/11f34af1-5f1b-4e42-9dfb-e91094081311" />

I am going to spell this out and maybe we can create a pattern.
- The Name Response is not a reading block on its own.
- The Visual Help Text is an all input reading block that starts with the bolded Name Response.

- [ ] **Reading Block Content:**
    - Lipids Animation. Remove wiggling animation of phospholipid bilayer forming the cell membrane.
    - Glucose Metabolism. Fade and remove glucose solutes to simulate conversion for cellular metabolism.

**Responses for toggle switch**
I am mirroring how it works for other toggle switches in the Preferences menu.
- [ ] On keyboard focus, voice the Name Response (the bolded text).
    - Lipids Animation.
    - Glucose Metabolism.

Note that for mouse/pointer input the name response is coming through the context response.

- [ ] On mouse activation and keyboard activation (i.e., active toggle) voice preferences context responses that includes all or some of the name response. Here it includes all of the name:
     - "Lipids Animation off." / "Lipids Animation on"
    -  "Glucose Metabolism on." / "Glucose Metabolism off."

**Comment by terracoda** [comment]
@brettfiedler, not sure why this was on hold. The Interactive Description part is required. The Voicing Part may not be required.

**Comment by terracoda** [comment]
Oh shoot. Oh well. We now have a target design.

**Comment by brettfiedler** [comment]
Reviewed together  with @terracoda . Thanks!!!
